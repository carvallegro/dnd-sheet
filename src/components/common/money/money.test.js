import React from 'react'
import { shallow } from 'enzyme'

import Money from './index'

describe('<Money />', () => {
  it('renders', () => {
    const wrapper = shallow(<Money />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with value', () => {
    const wrapper = shallow(<Money value={100} name="TC" />)
    expect(wrapper).toMatchSnapshot()
  })
})
