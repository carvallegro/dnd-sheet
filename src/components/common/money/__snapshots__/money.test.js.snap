// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Money /> renders 1`] = `
<styled.div
  justifySelf="inherit"
>
  <styled.h3 />
  <styled.div>
    <styled.p />
  </styled.div>
</styled.div>
`;

exports[`<Money /> renders with value 1`] = `
<styled.div
  justifySelf="inherit"
>
  <styled.h3>
    TC
  </styled.h3>
  <styled.div>
    <styled.p>
      100
    </styled.p>
  </styled.div>
</styled.div>
`;
